src/generator/wait-for-client-js.ts

```diff
@@ -41,7 +43,9 @@ export async function waitForClientJs(
                 String(await readFile(join(path, 'schema.prisma'))),
             );
 
-            if (generatedSchemaContents === currentSchema) {
+            if (useSchemaDir && generatedSchemaContents.includes(currentSchema)) {
```

Comment:
When `prismaSchemaFolder` is enabled, Prisma combines the source `*.prisma` files into a single `schema.prisma` file in the output.
Using `.includes()` might not be the best solution here, but I assume this check was placed here to ensure the file was generated - rather than actually checking the content of the file.

Reply:
Unfortunately the check was used to ensure that this package runs on the most up-to-date output of the Prisma client generator, not just making sure that the output exists. Nn `.includes()` might not be good enough.

I don't an obvious way of knowing if the client output is up to date (besides reading file mtime or ctime but doing so would be buggy) besides comparing the schema file contents. However, this is still an improvement, so I think it's still good to merge this. I'll file a related ticket about the limitation though.

Reply:
Filed in https://github.com/electrovir/prisma-frontend/issues/3